src/Makefile

```diff
@@ -167,17 +172,18 @@ check:
 checkpar:
 	@ \
         echo "Checking the library ..."; \
+        set -e; \
         (cd test; $(MAKE) all); \
         echo "Testing IJ ..."; \
-        (cd test; ./runtest.sh -tol 1.e-06 -mpi "$(CHECKRUN)" TEST_ij/solvers.sh); \
+        (cd test; ./runtest.sh -atol $(HYPRE_CHECKPAR_TOL) -mpi "$(CHECKRUN)" TEST_ij/solvers.sh); \
```

Comment:
@rfalgout I'm wondering if it is more appropriate to use `rtol` here and in the similar occurrences below. What do you think?

Reply:
It looks like we generally use atol in our AUTOTEST testing